llvm/lib/Target/X86/GISel/X86LegalizerInfo.h

```diff
@@ -22,16 +22,22 @@ class X86Subtarget;
 class X86TargetMachine;
 
 class X86LegalizerInfo : public LegalizerInfo {
-private:
-  /// Keep a reference to the X86Subtarget around so that we can
-  /// make the right decision when generating code for different targets.
-  const X86Subtarget &Subtarget;
-
```

Comment:
Leave this where it is?

Reply:
Yes. I've taken a look at `LegalizerInfo` and `AArch64LegalizerInfo`, they have private fields at the end of declaration. That's the reason I've decided it needs refactor. Apparently there is no strict rules about it. So, I've left it untouched.